views/forms/TextInput/index.jsx

```diff
@@ -1,19 +1,23 @@
+import React from 'react'
 import TextField from '@mui/material/TextField'
-import { Controller } from 'react-hook-form'
+import { Controller, useFormContext } from 'react-hook-form'
 
 const TextInput = (props) => {
-  const { name, control, onChange, ...rest } = props
+  const { name, onChange, ...rest } = props
+  const { control } = useFormContext()
```

Comment:
@Elkrival this change broke the admin page when a user logs in. I think we need the control from props.